src/components/TransactionBlock/PlrDaoStakingTransactionBlock.tsx

```diff
@@ -332,7 +326,7 @@ const PlrDaoStakingTransactionBlock = ({
         setIsLoadingAvailableOffers(false);
         if (!offers.length) return;
         const bestOffer: ExchangeOffer | undefined = offers?.find(
-          (offer) => offer.provider === swapServiceIdToDetails['Lifi'].title
+          (offer) => offer.provider === swapServiceIdToDetails['Lifi'].title,
```

Comment:
sorry but `offer.provider === swapServiceIdToDetails['Lifi'].title` doesn't make any sense because `offer.provider` is the id?

Reply:
Offer.Provider is the name here.